src/cecs277TermProject/RoachMotel.java

```diff
@@ -41,20 +41,21 @@ public RoachMotel(String name, int capacity, File outFile) throws FileNotFoundEx
 	// TODO: [Gust] Add a parameter for list of amenities
 	public void checkIn(RoachColony roachColony) {
 		// TODO: [Tyler] Implement logging this event here
+		// Logging should be mostly finished; just need room methods to be finished - Tyler
```

Comment:
Which room methods are you waiting for? I think you only need the `getCost()` method (which already merged in master). Gust will take care of implementing Decorator pattern, which include adding amenities, but the interface should not change. Am I missing something?

Reply:
I only saw room number in the Room.toString().  If that's gonna print out amenities too I can get rid of my placeholder text and it should be working completely then.